packages/address/src/tests/AddressFormatter.test.ts

```diff
@@ -137,6 +137,13 @@ describe('AddressFormatter', () => {
 
       expect(fetch.calls()).toHaveLength(2);
     });
+
+    it('allows the optional parameter `includeHiddenZones:`.', async () => {
+      const addressFormatter = new AddressFormatter('en');
+      await addressFormatter.getCountries({includeHiddenZones: true});
+
+      expect(fetch.calls()).toHaveLength(1);
```

Comment:
This test doesn't seem very useful.  It should test that `getCountries` returns some new additional countries that were not available when you call getCountries without `includeHiddenZones`. It sounds like this can be done by following @trishrempel's suggestion to add support for `includeHiddenZones` to the address mocks.


We should also test that the the regular getCountries omits that hidden country.

Reply:
That's a fair point.

I've revised the mocks to return an extra zone, `DH`, for country `IN` India, and put new tests in `AddressFormatter.test.ts` that check for its presence/absence:
 - returns hidden zones when `includeHiddenZones` is `true`
 - excludes hidden zones when `includeHiddenZones` is `false`
 - excludes hidden zones when `includeHiddenZones` is not passed

https://github.com/Shopify/quilt/pull/2522/commits/d17b6386b1437eb861b2b44e46a472652352edfa#diff-03ac05a5590924ec807cb0030e4ba5d5ac68662f4726dd8470be77887228a0a9